### DISCUSSION TITLE
[QUESTION]: noExternal dev vs prod

### DISCUSSION THREAD

**Discussion by simplecommerce** [vikejs/vike#1422] [Discussion] [answered]
Hi,

I have a question in regards to passing noExternal when using ssr.
I am using latest version of vike and vite on Node v16.

I noticed that this works in production but not in dev.

`/@mui\/.*/` 

In dev, it doesn't seem to work and I have to manually add the following, for it to load properly.

```
	"@mui/base",
	"@mui/icons-material",
	"@mui/material",
	"@mui/utils",
	"@mui/lab",
	"@mui/x-date-pickers",
	"@mui/system",
	"@mui/styled-engine",
```

Is this normal behavior?
I don't know if its vike or vite, I figured i'd ask.

Thanks.

**Comment by brillout** [comment, marked as answer]
It's a Vite question. Let me know what you find and/or if you create a Vite issue for this. I've also been wondering whether wildcard `ssr.noExternal` are possible.
